chapter_computer-vision/rcnn_vn.md

```diff
@@ -269,7 +278,10 @@ as well as the binary category and bounding box predictions for the anchor boxes
 Finally, the region proposal network can learn how to generate high-quality proposed regions, which reduces the number of proposed regions while maintaining the precision of object detection.
 -->
 
-*dịch đoạn phía trên*
+Đáng lưu ý rằng, vì là một phần của mô hình Faster R-CNN, nên mạng đề xuất khu vực được huấn luyện cùng với phần còn lại trong mô hình.
+Ngoài ra, các chức năng đối tượng trong Faster R-CNN bao gồm lớp và khung chứa dự đoán trong phát hiện đối tượng,
+cũng như là phân loại nhị phân và dự đoán các khung chứa cho các khung neo nằm trong mạng đề xuất khu vực.
+Sau cùng, mạng đề xuất khu vực có thể học được cách sinh ra những vùng đề xuất có chất lượng cao, nhằm giảm đi số lượng vùng đề xuất trong khi vẫn giữ được precision của việc phát hiện đối tượng
```

Comment:
```suggestion
Sau cùng, mạng đề xuất vùng có thể học được cách sinh ra những vùng đề xuất có chất lượng cao, nhằm giảm đi số lượng vùng đề xuất trong khi vẫn giữ được độ chính xác khi phát hiện vật thể.
```
precision này không cần phải phân biệt với accuracy trong bài toán phân loại nên mình có thể dịch là `độ chính xác` luôn.